invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/dropdown.overrides

```diff
@@ -97,8 +95,18 @@
       font-weight: normal;
 
     }
+
+    .dropdown.icon {
+      padding-top: 0.25em;
```

Comment:
this should not be necessary if you use `flex` class on the parent element (if possible)

Reply:
I could not manage to do it with `flex`, the icon inside the SUI Dropdown would not align to the end of the flexbox.